DigitalLearningSolutions.Data.Tests/DataServices/UserDataServiceTests.cs

```diff
@@ -5,6 +5,7 @@
     using DigitalLearningSolutions.Data.DataServices;
```

Comment:
Could we add this page to the automated accessibility tests if it's not there already? 🙏